substrate/frame/src/lib.rs

```diff
@@ -202,10 +202,17 @@ pub mod prelude {
 	/// Dispatch types from `frame-support`, other fundamental traits
 	#[doc(no_inline)]
 	pub use frame_support::dispatch::{GetDispatchInfo, PostDispatchInfo};
-	pub use frame_support::traits::{
-		Contains, EitherOf, EstimateNextSessionRotation, Everything, IsSubType, MapSuccess,
-		NoOpPoll, OnRuntimeUpgrade, OneSessionHandler, RankedMembers, RankedMembersSwapHandler,
-		VariantCount, VariantCountOf,
+	pub use frame_support::{
+		defensive, defensive_assert, impl_ensure_origin_with_arg_ignoring_arg,
+		storage::KeyLenOf,
+		traits::{
+			BalanceStatus, Contains, Currency, Defensive, DefensiveOption, EitherOf, EnsureOrigin,
+			EnsureOriginWithArg, EstimateNextSessionRotation, ExistenceRequirement::AllowDeath,
+			Imbalance, Instance, IsSubType, MapSuccess, NoOpPoll, OnRuntimeUpgrade, OnUnbalanced,
+			OneSessionHandler, Randomness, RankedMembers, RankedMembersSwapHandler,
+			ReservableCurrency, StorageVersion, UncheckedOnRuntimeUpgrade,
+		},
+		PalletId, migrations::VersionedMigration,
```

Comment:
Both `UncheckedOnRUntimeUpgrade` and `VersionedMigration` seems to better go into a new prelude: `migration_prelude` or something like this.